operations/system_management_health/Troubleshoot_Prometheus_Alerts.md

```diff
@@ -44,5 +45,36 @@ Alerts for CPUThrottlingHigh on CFS services such as cfs-batcher and cfs-trust c
 
 The CephNetworkPacketsDropped alert does not necessarily indicate there are packets being dropped on an interface on a storage node. In a future release this alert will be renamed to be more generic. If this alert fires, inspect the IP address in the details of the alert to determine the node in question (can be storage, master or worker node). If the interface in question is determined to be healthy, this alert can be ignored.
 
+<a name="cephmgrmissing"></a>
+## CephMgrIsAbsent and CephMgrIsMissingReplicas
 
+If the CephMgrIsAbsent and/or CephMgrIsMissingReplicas alerts fire, use the following steps to ensure the `prometheus` module has been enabled for `Ceph`. The following steps should be executed on ncn-s001:
 
+```bash
```

Comment:
1. List the enabled modules and look for `prometheus` in the returned output.
    ```bash
    ncn-s001# ceph mgr module ls | jq '.enabled_modules'
    [
      "cephadm",
      "iostat",
      "restful"
    ]
    ```
    
    If `prometheus` is missing from the output, enable it with the following command:
    
    ```bash
    ncn-s001# ceph mgr module enable prometheus
    ```